src/test/java/seedu/address/model/person/EmailTest.java

```diff
@@ -13,19 +13,12 @@ public void constructor_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new Email(null));
     }
 
-    @Test
-    public void constructor_invalidEmail_throwsIllegalArgumentException() {
-        String invalidEmail = "";
```

Comment:
Would it be better to change the invalidEmail to something else instead of removing this test?